<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pedido_de_compra</class>
 <widget class="QDialog" name="pedido_de_compra">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>138</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>Tipo de entrega</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>14</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QRadioButton" name="rb_entrega_retirar">
       <property name="text">
        <string>A retirar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_entrega_frete">
       <property name="text">
        <string>Entrega com frete</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>191</y>
     <width>741</width>
     <height>371</height>
    </rect>
   </property>
   <property name="title">
    <string>Itens do pedido</string>
   </property>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>123</y>
      <width>721</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>527</x>
      <y>14</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Tipo desconto</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QRadioButton" name="rb_tipo_desc_moeda">
        <property name="text">
         <string>$</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_tipo_desc_porcent">
        <property name="text">
         <string>%</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>721</width>
      <height>45</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Data de emissão</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEdit_pedido_data_emissao"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Previsão de entrega</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEdit_pedido_prev_entrega"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_14">
       <item>
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Quantidade total de itens</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_quant_total_itens">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>250</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_15">
       <item>
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Valor total do pedido</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_valor_total_ped">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>634</x>
      <y>20</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_13">
     <item>
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Valor desconto</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txt_valor_desconto"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>511</width>
      <height>43</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Código produto</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_prod_cod"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Quantidade</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_prod_qtde"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Estoque total disponível</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_prod_estoq_disp">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_12">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Preço</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_prod_preco">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>721</width>
      <height>43</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_16">
       <item>
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Nome Produto</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_nome_produto">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QPushButton" name="btn_item_add">
         <property name="text">
          <string>Adicionar Item</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_item_edit">
         <property name="text">
          <string>Editar Item</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_item_excluir">
         <property name="text">
          <string>Excluir Item</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_finalizar_ped">
   <property name="geometry">
    <rect>
     <x>658</x>
     <y>568</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Finalizar pedido</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_novo_ped">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Novo Pedido</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>81</y>
     <width>731</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Código cliente</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_cod_cli"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Nome do cliente</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_nome_cli"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>CNPJ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_cnpj_cli"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Tipo de pagamento</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cb_tipo_pag_cli"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>138</y>
     <width>531</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Código transportadora</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_cod_transp"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Valor do frete</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_valor_frete"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_tipo_consulta">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Tipo de Consulta</string>
   </property>
   <widget class="QRadioButton" name="rb_cod_cli">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Código</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_nome_cli">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Nome</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_cnpj">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>CNPJ</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>135</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_17">
    <item>
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Código Pedido</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_cod_ped">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
